Update live events support for AzureDevOps (#2185)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/azure-devops/installation.md

The token should either have `admin` permissions, or `read` permissions for each of the supported resources you want to ingest into Port.

> ## BaseUrl & Webhook Configuration

Reviewer: ```suggestion
## BaseUrl & webhook configuration
```

:::warning AppHost deprecation
**`integration.config.appHost` is deprecated**: Please use `baseUrl` for webhook URL settings instead.
:::

> The `baseUrl` parameter enables real-time updates from Azure DevOps to Port. If not provided:

Reviewer: ```suggestion
The `baseUrl` parameter enables real-time updates from Azure DevOps to Port.  
If not provided:
```

- The integration will still function normally

> - You'll need to use [`scheduledResyncInterval`](https://ocean.getport.io/develop-an-integration/integration-configuration/#scheduledresyncinterval---run-scheduled-resync) for updates

Reviewer 2: ```suggestion
- You should use [`scheduledResyncInterval`](https://ocean.getport.io/develop-an-integration/integration-configuration/#scheduledresyncinterval---run-scheduled-resync) to configure updates at a set interval.
```

The token should either have `admin` permissions, or `read` permissions for each of the supported resources you want to ingest into Port.

### AppHost & listening to hooks
## BaseUrl & webhook configuration

:::tip
The `appHost` parameter is used specifically to enable the real-time functionality of the integration.

If it is not provided, the integration will continue to function correctly. In such a configuration, to retrieve the latest information from the target system, the [`scheduledResyncInterval`](https://ocean.getport.io/develop-an-integration/integration-configuration/#scheduledresyncinterval---run-scheduled-resync) parameter has to be set, or a manual resync will need to be triggered through Port's UI.
:::warning AppHost deprecation
**`integration.config.appHost` is deprecated**: Please use `baseUrl` for webhook URL settings instead.
:::

In order for the Azure Devops integration to update the data in Port on every change in the Azure Devops repository, you need to specify the `appHost` parameter.

> The `appHost` parameter should be set to the `url` of your Azure Devops integration instance. In addition, your Azure Devops instance (whether it is Azure Devops SaaS or a self-hosted version of Azure Devops) needs to have the option to send webhook requests to the Azure Devops integration instance, so please configure your network accordingly.

Reviewer 3: keep this, change `appHost` hints to `baseUrl`
